### ISSUE TITLE
TextEditor.getLastVisibleScreenRow is deprecated.

### ISSUE THREAD

**Issue by fkautz** [atom/vim-mode#632]
Reporting this on behalf of the plugin. I did an issue search and did not find a duplicate.

This is now a view method. Call TextEditorElement::getLastVisibleScreenRow instead.
```
TextEditor.getLastVisibleScreenRow (/usr/share/atom/resources/app/src/text-editor.js:2620:9)
MoveToBottomOfScreen.getDestinationRow (/home/fkautz/.atom/packages/vim-mode/lib/motions/general-motions.coffee:367:28)
```

**Comment by FXuZ** [comment]
Similarly `TextEditor.getFirstVisibleScreenRow` is also deprecated. The warning given by atom is as following:

This is now a view method. Call `TextEditorElement::getFirstVisibleScreenRow` instead.
```
TextEditor.getFirstVisibleScreenRow (/usr/local/share/atom/resources/app.asar/src/text-editor.js:2613:9)
ScrollHalfScreenDown.Scroll (/home/gfrank/.atom/packages/vim-mode/lib/scroll.coffee:7:21)
```

**Comment by daniloisr** [comment]
There is already a PR for it https://github.com/atom/vim-mode/pull/630 =)

**Comment by maxbrunsfeld** [comment]
Fixed via #630.
